src/migrations/20250805152422-backfill-lifecycle-trends.js

```diff
@@ -0,0 +1,91 @@
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+            DELETE FROM lifecycle_trends;
+
+            WITH week_ranges AS (
```

Comment:
Generate week ranges, This basically generates 52 weeks in format 2025-31, July 27-Aug 03.